packages/node_modules/nav-frontend-tabs/_tabs.sample.js

```diff
@@ -13,7 +13,7 @@ export default generateSample({
         ],
         onChange: () => {}
     },
-    modifierNames: ['kompakt', 'manuell'],
+    modifierNames: ['kompakt'],
```

Comment:
Burde den ha den nye (renamede) parameteren inn?

Reply:
Hm, hvordan gjør vi det med modifiers hvor default er `true`?

Reply:
🤔 vet ikke. Det må vel implementeres isåfall, men det kan være på en egen PR om noe. Approver nå.